src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -30,12 +32,36 @@ import { default as useSelector, default as useTypedSelector } from 'src/redux/u
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
 import { saveProfilePicture } from 'src/utils/image'
+import Logger from 'src/utils/Logger'
 import { useAsyncKomenciReadiness } from 'src/verify/hooks'
+import { Statsig } from 'statsig-react-native'
 
 type Props = StackScreenProps<StackParamList, Screens.NameAndPicture>
 
+const getExperimentParams = () => {
```

Comment:
Side comment: This seems ripe for a helper function to share code for getting params from experiment layers. Not too concerned if its done now or later when we have other statsig experiments.

Reply:
Yeah agreed, I lean towards doing that when we have other experiments.